org.eclipse.jdt.core.manipulation/core extension/org/eclipse/jdt/internal/corext/refactoring/refactoring.properties

```diff
@@ -1343,6 +1343,27 @@ IntroduceIndirectionRefactoring_call_warning_anonymous_cannot_qualify=The declar
 IntroduceIndirectionRefactoring_cannot_update_binary_target_visibility=A visibility update is needed for the member ''{0}'' or an enclosing type, but the member is not writable. Refactoring will introduce compile errors.
 IntroduceIndirectionRefactoring_cannot_run_without_intermediary_type=Cannot run Introduce Indirection without a target type.
 
+#--- Make Static
+MakeStaticRefactoring_name=Make Static
+MakeStaticRefactoring_not_available_on_this_selection=Make Static is not available on this selection. Select a method invocation or a method declaration.
+MakeStaticRefactoring_not_available_on_annotation=Make Static is not supported on annotation types.
+MakeStaticRefactoring_not_available_for_local_or_anonymous_types=Make Static is not supported on methods inside local or anonymous types.
+MakeStaticRefactoring_not_available_for_constructors=Make Static is not available for constructors.
+MakeStaticRefactoring_checking_activation=Checking activation of Make Static...
+MakeStaticRefactoring_not_available_for_super_method_invocations=Make Static is not available for SuperMethodInvocations.
+MakeStaticRefactoring_parameter_name_already_used=Parameter with name ''{0}'' already exists in method declaration.
+MakeStaticRefactoring_duplicate_method_signature=Method with the same signature as the refactored method already exists.
+MakeStaticRefactoring_method_already_static=Selected method is already static.
+MakeStaticRefactoring_method_is_overridden_in_subtype=Selected method is overridden in subtype. 
+MakeStaticRefactoring_hiding_method_of_parent_type=Make Static is not available. Method of parent type would be hidden after refactoring.
+MakeStaticRefactoring_not_available_for_recursive_methods=Make Static is not available for recursive methods.
+MakeStaticRefactoring_not_available_for_parametrized_methods=Make Static is not available for parametrized methods.
+MakeStaticRefactoring_not_available_for_wildCardTypes_as_bound=Make Static is not available if class has WildCardTypes as bounds of its type parameters.
+MakeStaticRefactoring_not_available_for_method_references=Make Static is not available if method references use selected method.
+MakeStaticRefactoring_explicit_super_method_invocation=The method contains explicit invocation of a superclass method using 'super' and will lead to behavior changes and break the class hierarchy.
+MakeStaticRefactoring_selected_method_uses_super_field_access=The method contains explicit access of a superclass field using 'super' and will lead to behavior changes and break the class hierarchy.
```

Comment:
It now says "Applying the refactoring would lead to behavior changes". The class hierarchy would not be broken by performing the refactoring. But the semantics would change because an instance of the subclass would be used instead of the super keyword. So it is right to block this case with an error message but the message is now less confusing.